api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -117,13 +118,13 @@ public static Module loadModule(File moduleFile, Boolean replaceIfExists) throws
 	 * 
 	 * @param module
 	 * @param replaceIfExists unload a module that has the same moduleId if one is loaded already
-	 * <strong>Should</strong> load module if it is currently not loaded
-	 * <strong>Should</strong> not load module if already loaded
-	 * <strong>Should</strong> always load module if replacement is wanted
-	 * <strong>Should</strong> not load an older version of the same module
-	 * <strong>Should</strong> load a newer version of the same module
-	 * @return module the module that was loaded or if the module exists already with the same
-	 *         version, the old module
+	 *            <strong>Should</strong> load module if it is currently not loaded
+	 *            <strong>Should</strong> not load module if already loaded <strong>Should</strong>
```

Comment:
I guess this was just a formatting change.

Reply:
There is a one line change in the actual code, `null` was replaced with 'new Contexts()` in a method call. The other changes are probably due to formatting.